code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm

```diff
@@ -0,0 +1,52 @@
+GLOBAL_LIST_INIT(electrolyzer_reactions, electrolyzer_reactions_list())
+
+/*
+ * Global proc to build the electrolyzer reactions list
+ */
+/proc/electrolyzer_reactions_list()
+	. = list()
+	for(var/reaction_path in subtypesof(/datum/electrolyzer_reaction))
+		var/datum/electrolyzer_reaction/reaction = new reaction_path()
+
+		.[reaction.id] = reaction
+
+/datum/electrolyzer_reaction
+	var/list/requirements
+	var/name = "reaction"
+	var/id = "r"
+
+/datum/electrolyzer_reaction/proc/react(turf/location, working_power, delta_time)
+
+/datum/electrolyzer_reaction/h2o_conversion
+	name = "H2O Conversion"
+	id = "h2o_conversion"
+	requirements = list(
+		/datum/gas/water_vapor = MINIMUM_MOLE_COUNT
+	)
+
+/datum/electrolyzer_reaction/h2o_conversion/react(turf/location, working_power, delta_time)
+
+	var/datum/gas_mixture/air_mixture = location.return_air()
+
+	air_mixture.assert_gases(/datum/gas/water_vapor, /datum/gas/oxygen, /datum/gas/hydrogen)
+	var/proportion = min(air_mixture.gases[/datum/gas/water_vapor][MOLES], (5 * delta_time * (working_power ** 2)))
+	air_mixture.gases[/datum/gas/water_vapor][MOLES] -= proportion * 2
+	air_mixture.gases[/datum/gas/oxygen][MOLES] += proportion
+	air_mixture.gases[/datum/gas/hydrogen][MOLES] += proportion * 2
+
+/datum/electrolyzer_reaction/nob_conversion
+	name = "Hypernob conversion"
+	id = "nob_conversion"
+	requirements = list(
+		/datum/gas/hypernoblium = MINIMUM_MOLE_COUNT,
+		"MAX_TEMP" = 150
+	)
+
+/datum/electrolyzer_reaction/nob_conversion/react(turf/location, working_power, delta_time)
+
+	var/datum/gas_mixture/air_mixture = location.return_air()
+
+	air_mixture.assert_gases(/datum/gas/hypernoblium, /datum/gas/antinoblium)
+	var/proportion = min(air_mixture.gases[/datum/gas/hypernoblium][MOLES], (3 * delta_time * (working_power ** 2)))
+	air_mixture.gases[/datum/gas/hypernoblium][MOLES] -= proportion
+	air_mixture.gases[/datum/gas/antinoblium][MOLES] += proportion * 0.5
```

Comment:
Being honest with me, how "good" is this?

Reply:
is good for an alternative to make antinoblium, but the rates are around 6-10 moles per process() tick of antinoblium, so is good to have for an early HFR hyper-anti fuel or small scale antinob SM but other than that not much (unless i'm not seeing something else)